generator/src/main/java/com/algorand/sdkutils/utils/Tools.java

```diff
@@ -0,0 +1,77 @@
+package com.algorand.sdkutils.utils;
+
+import java.util.StringTokenizer;
+
+public class Tools {
+
+    // formatComment formats the comment by breaking lines, and incorporating 
+    // embedded formatting inside the comment. 
+    // If the comment is embedded inside another comment, full == false
+    public static String formatComment(String comment, String tab, boolean full) {
```

Comment:
We should do the HTML encoding, apache-commons has a `StringUtils.encodeHtml` helper that should be able to do this for us.

Reply:
This can be future enhancement?

Reply:
yes